docs/api-reference/contexts.md

```diff
@@ -225,31 +255,47 @@ invoke<T>(targetClass: T, workflowUUID?: string): InvokeFuncs<T>
 ```
 
 Invoke transactions and communicators.
-To invoke other workflows, use [`childWorkflow`](#workflowctxtchildworkflow).
+To invoke other workflows, use [`invokeChildWorkflow`](#workflowctxtinvokechildworkflow) or [`startChildWorkflow`](#workflowctxtstartchildworkflow).
 
-The syntax for invoking function `foo` in class `Bar` with argument `baz` is:
+The syntax for invoking function `fn` in class `Cls` with argument `arg` is:
 
 ```typescript
-workflowCtxt.invoke(Bar).foo(baz)
+const output = await workflowCtxt.invoke(Cls).fn(arg)
 ```
 
-Note that the DBOS runtime will supply a context to invoked functions.
+You don't need to supply a context to the invoked function&#8212;the DBOS Transact runtime does this for you.
+
+#### `workflowCtxt.invokeChildWorkflow`
+
+```typescript
+invokeChildWorkflow<T extends any[], R>(wf: Workflow<T, R>, ...args: T): Promise<R>
+```
 
-#### `workflowCtxt.childWorkflow`
+Invoke a child workflow and wait for it to complete, returning its result.
+The syntax for invoking workflow `wf` in class `Cls` with argument `arg` is:
 
 ```typescript
-childWorkflow<T extends any[], R>(wf: Workflow<T, R>, ...args: T): Promise<WorkflowHandle<R>>
+const output = await ctxt.invokeChildWorkflow(Cls.wf, arg)
 ```
 
-Invoke another workflow.
-This returns a [workflow handle](./workflow-handles) for the new workflow.
+You don't need to supply a context to the invoked child workflow&#8212;the DBOS Transact runtime does this for you.
 
-The syntax for invoking workflow `foo` in class `Bar` with argument `baz` is:
+#### `workflowCtxt.startChildWorkflow`
 
 ```typescript
-const workflowHandle = await ctxt.childWorkflow(Bar.foo, baz)
+startChildWorkflow<T extends any[], R>(wf: Workflow<T, R>, ...args: T): Promise<WorkflowHandle<R>>
 ```
 
+Durably start a child workflow and return a [handle](./workflow-handles.md) to it but do not wait for it to complete.
+This method resolves as soon as the handle is safely created; at this point the workflow is guaranteed to [run to completion](../tutorials/workflow-tutorial.md#reliability-guarantees).
```

Comment:
safely -> durably?  And maybe remove "Durably" above, as there's no other way we do it.